content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Create Azure DevOps Work Item for Findings"
+linkTitle: "Create Azure DevOps Work Item for Findings"
+weight: 40
+---
+
+## Configure Azure DevOps Integration for New Findings
+
+To set up Azure DevOps to create Work Items for new findings, follow these two main steps:
+
+1. [Create an Azure DevOps connection](#create-an-azure-devops-connection).
+2. [Create a new Azure DevOps recipe](#create-a-new-azure-devops-recipe).
+
+### Create a New Folder for Your Azure DevOps Recipes
+
+Before you begin, it's a good idea to create a new folder for all your future Azure DevOps recipes.
+
+Navigate to the Projects tab in the Integration Builder.
+
+Click the "Create" button at the top right, then select "Folder".
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/01-create-folder.png" alt="Create a new folder" %}}
+
+Name the new folder and choose its location. Then click "Create folder".
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/02-create-folder-modal.png" alt="Folder details" %}}
+
+You should now see the new folder in the project.
+
+### Create an Azure DevOps Connection
+
+Click the "Create" button at the top right, then select "Connection".
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/03-create-connection.png" alt="Create a new connection" %}}
+
+Search for the Azure DevOps connection and click on it.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/04-select-azure-devops-connection.png" alt="Select Azure DevOps connection" %}}
+
+Fill in the connection details: name, recipe location, and select **Personal Access Token** as the OAuth 2.0 grant type. Set the API version to **7.0** and specify the Azure DevOps organization name, the email address of the user to impersonate, and the personal access token.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/05-connection-details.png" alt="Connection details" %}}
```

Comment:
Please show a dummy email or pixelate it.

Reply:
https://github.com/cobalthq/cobalt-product-public-docs/pull/562/commits/754c37f062f10de2500eeb8f5ef8ff14c2c5644f